src/styles/main.scss

```diff
@@ -1,3 +1,70 @@
+@import 'utils/variables';
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+  width: 710px;
```

Comment:
these comments were not fixed:

![image](https://github.com/mate-academy/layout_calendar/assets/86923089/b61ebde1-3b2f-4c7b-beff-b5ebc12e6a92)

About width calculation: you have all variables for calculations - create one more for calculated width